contents/blog/best-plausible-alternatives.mdx

```diff
@@ -0,0 +1,459 @@
+</ComparisonTable>
+
+### How popular is PostHog?
+
+According to [BuiltWith](https://trends.builtwith.com/analytics/PostHog), as of October 2024, 5,330 of the top 1 million websites deploy PostHog, significantly more than the 3,500 using Plausible.
+
+### Why do companies use PostHog?
+
+According to [reviews on G2](https://www.g2.com/products/posthog/reviews), companies use PostHog because:
+
+1. **It replaces multiple tools:** PostHog can replace Plausible (web analytics), [Amplitude](/blog/posthog-vs-amplitude) (product analytics), and [Fullstory](/blog/posthog-vs-fullstory) (session replay). This simplifies workflows and ensures product data is all in one place.
+
+2. **Pricing is transparent and scalable:** Reviewers appreciate how PostHog's pricing scales as they grow. There's a [generous free tier](/pricing) they can use forever. Companies eligible for [PostHog for Startups](/startups) also get $50k in additional free credits.
+
+3. **They need a complete picture of users:** PostHog includes every tool necessary to understand users and improve products. This means creating funnels to track conversion, watching replays to see where users get stuck, testing solutions with A/B tests, and gathering feedback with user surveys.
+
+> #### Bottom line
+> Because PostHog is free, has web analytics and more, and still has the privacy and compliance features Plausible offers, it makes for a great alternative, especially for developers and startups.
+
+## 2. Fathom
+
+- **Founded:** 2018
+- **Similar to:** Plausible, Google Analytics
+- **Typical users:** Founders and content teams
+- **Typical customers:** Privacy-conscious, content-focused websites
+
+![Fathom](https://res.cloudinary.com/dmukukwp6/image/upload/Clean_Shot_2024_10_02_at_16_12_06_c41b89ea2f.png)
+
+### What is Fathom?
+
+Fathom is a privacy-focused [Google Analytics alternative](/blog/ga4-alternatives). It is a simple web analytics tool that captures details on traffic, sessions, referrers, sources, and campaigns. It does this while helping users stay compliant and avoid cookies.
+
+### Key features
+
+- 🧑‍🎨 **Simple web analytics:** Fathom provides a simple, GA-like experience to monitor your website and get an overview of your traffic.
+
+- 🍪 **Cookieless:** Avoid cookie banners and track your site without using cookies.
+
+- 📊 **Marketing metrics:** Track the sources, devices, browsers, locations, UTMs, and more for your traffic. 
+
+- ⭐ **Conversions:** Track custom events and analyze them as conversions towards a goal.
+
+### How does Fathom compare to Plausible?
+
+Feature-wise, Plausible and Fathom are nearly identical and this shows on their website and design. Looking closer, Fathom is missing some of the compliance features Plausible offers, like [open source](/blog/best-open-source-analytics-tools) code and EU hosting.
```

Comment:
```suggestion
Feature-wise, Plausible and Fathom are nearly identical. Looking closer, however, Fathom is missing some of the compliance features Plausible offers, like [open source](/blog/best-open-source-analytics-tools) code and EU hosting.
```

Reply:
We might need to explain what Fathom does instead of EU-hosting here. iirc, it was some kind of routing / residency thing going on, but I can't recall the specifics.